10up/wp-newrelic
# Add Initial E2E tests

### Description of the Change
This PR introduces an initial E2E setup with basic E2E tests. Please note that it does not include extensive E2E tests to cover the full functionality of the plugin. Additional effort will be required to configure or simulate New Relic for comprehensive verification.

Closes #55

### How to test the Change
1. Run `npm install`
2. Run `npm run env:start && npm run cypress:run`
3. Verify all E2E tests are passing.
4. Verify E2E tests are passing on GH action.

### Changelog Entry
> Added - Initial E2E tests setup and Added basic E2E tests

### Credits
Props @mehul0810 (Thanks for raising the initial PR) @iamdharmesh @jayedul

### Checklist:
